### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dhritzkiv**
Fixes some code, language, and formatting typos.

I reworded the `Caveats` section, but I'm not sure if that's what was ultimately meant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ That's it! You are good to go.
 
 ## Caveats
 
-**IMPORTANT:** Be aware that `server-side` includes every first render of your application.
-Your secret tokens won't be included anywhere **except you use them **
+**IMPORTANT:** Be aware that `server` is included on the first render of your application.
```
Comment (TheAlexLichter) [inline README.md]:
```suggestion
**IMPORTANT:** Be aware that `server-side` means on **every** first render of your application.
```

That's what I actually meant :P

File: README.md [hunk 2]
```diff
@@ -54,8 +54,8 @@ That's it! You are good to go.
...
+Your secret tokens won't be included anywhere **except where you use them **
```
Comment (TheAlexLichter) [inline README.md]:
```suggestion
Your secret tokens won't be included anywhere **except where you use them**
```

**Review by TheAlexLichter — CHANGES_REQUESTED** [review]
Thanks so far! Left you some comments ;)

File: README.md (hunk 2 above)
Comment (dhritzkiv) [reply README.md]: Ah, okay. I misunderstood how `server-side` was used, since it looked like it was referring to a variable/value.

**Comment by ricardogobbosouza** [comment]
Nice! @manniL can merged?

**Comment by TheAlexLichter** [comment]
yup,thx!

**Pull request merged**